method_mapper.py

```diff
@@ -1,12 +1,33 @@
-from views import get_all_amenities, get_all_amenity_types, get_amenity_by_id, get_amenity_type_by_id, get_all_blogs, get_single_blog, get_blogs_by_user_id, get_blogs_by_park_id, create_blog, update_blog, get_all_wildlife, get_single_wildlife_type, get_wildlife_by_park_id, create_wildlife, update_wildlife, delete_wildlife, get_all_natural_attractions, get_single_natural_attraction, get_natural_attraction_by_park_id, create_natural_attraction, update_natural_attraction, delete_natural_attraction
+from views import get_all_amenities, get_all_amenity_types, get_amenity_by_id, get_amenity_type_by_id, get_all_blogs, get_single_blog, get_blogs_by_user_id, get_blogs_by_park_id, create_blog, update_blog, get_all_wildlife, get_single_wildlife_type, get_wildlife_by_park_id, create_wildlife, update_wildlife, delete_wildlife, get_all_natural_attractions, get_single_natural_attraction, get_natural_attraction_by_park_id, create_natural_attraction, update_natural_attraction, delete_natural_attraction, get_all_parks, get_all_photos, get_all_user_favorites, get_all_users, get_single_park, get_single_photo, get_user_by_id, get_user_favorite_by_id, create_user, create_user_favorite, delete_user_favorite, delete_user, delete_park_amenity, delete_amenity_type, create_park_amenity, create_amenity_type, update_user, update_park_amenity, update_amenity_type
```

Comment:
Not an issue with your imports, just curious... Should we be breaking down the imports different? pylint says that we have too many imports in one line, and we obviously have a lot of imports in that one line. I'm not exactly sure what would be more efficient.

Reply:
We could group them in parenthesis and break them into multiple lines if we want. The nicest thing about this method is that all functions related to specific tables are on 1 line so there's less conflict at merge